src/applications/terms-of-use/touData.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
+import IdentityPhone from 'platform/user/authentication/components/IdentityPhone';
```

Comment:
The content in here is approved by OGC (Office of General Counsel), sitewide & CAIA so I would recommend just importing the `va-telephone` + `CONTACTS` from the web component and not our Identity component here.